packages/core/src/components/hotkeys/hotkey.tsx

```diff
@@ -50,6 +51,14 @@ export interface IHotkeyProps {
     group?: string;
 
     /**
+     * The platform type on which the application is running. This informs how keys will be named
+     * within the hotkey dialog. For example, on `PlatformType.MAC`, the hotkey dialog will refer to
+     * the "meta" key as <kbd>Cmd</kbd>, while on Windows, the hotkey dialog will refer to the
+     * "meta" key as <kbd>Ctrl</kbd>.
+     */
+    platformType?: PlatformType;
```

Comment:
can we put this at a higher level since it's basically an application-wide constant, instead of needing to set it on each individual `Hotkey` in your app?
might be a reasonable use case for React context...
perhaps `hotkeysDialogProps`?